Ukol_2.py

```diff
@@ -0,0 +1,26 @@
+import requests 
+import json
+
+try:
+    response = requests.get("https://cat-fact.herokuapp.com/facts/random", params={"animal_type": "cat" ,"amount": 10}, timeout=0.001)
+    data = response.json()
+
+    fakty = []
+    counter = 1
+
+    for fact in data:
+        fakty.append(f"{counter}. {fact["text"]}")
+        counter += 1

```

Comment:
```suggestion

    for index, fact in enumerate(data):
        fakty.append(f"{index}. {fact["text"]}")

```
Použití counter je dobre, existuje funkce enumerate která vlastně dělá něco podobné, vrací indexy :) myslím že má i nepovinný argument na začátek aby to nebylo od nuly